### PR TITLE
Fix undefined sourcePath in debugger

### PR DESCRIPTION
**Pull request by MickdeGraaf**
Fixes: https://github.com/trufflesuite/truffle-debugger/issues/51

### CODE REVIEW

File: packages/truffle-debugger/lib/session/sagas/index.js [hunk 1]
```diff
@@ -15,98 +15,104 @@ import * as web3 from "lib/web3/sagas";
...
-    yield *solidity.addSource(source, sourcePath, ast);
+  for (let i = 0; i < sources.length; i++) {
+    const sourceData = sources[i];
+    if (sourceData !== undefined && sourceData !== null) {
+      yield* solidity.addSource(
+        sourceData.source,
+        sourceData.sourcePath,
+        sourceData.ast
+      );
+    }
```
Comment (gnidan) [inline packages/truffle-debugger/lib/session/sagas/index.js]: But why is it undefined?

Comment (MickdeGraaf) [reply packages/truffle-debugger/lib/session/sagas/index.js]: Something to do with interfaces I believe. When importing something goes wrong with them. I dont know. Saw it somewhere as a fix and there was no fix merged anywhere so I created a pull request.

Comment (hjaltman) [reply packages/truffle-debugger/lib/session/sagas/index.js]: (Btw, I'm guessing most of these nonfunctional changes due to that linter we're now using?  Man, this is going to make reviewing PR's a bit of a pain for a while...)

Comment (hjaltman) [reply packages/truffle-debugger/lib/session/sagas/index.js]: After looking deeper at the code... I'm still pretty confused as to why this is occurring!  I could maybe see it if the `files` option was used, but it doesn't seem like the cases causing the problem involve that.

**Pull request merged**